google/cloud/bigquery/v2/minimal/internal/job_request_test.cc

```diff
@@ -177,6 +177,32 @@ TEST(ListJobsRequestTest, OutputStream) {
   EXPECT_EQ(expected, os.str());
 }
 
+TEST(GetJobRequest, DebugString) {
+  GetJobRequest request("test-project-id", "test-job-id");
+  request.set_location("test-location");
+  std::string expected = absl::StrCat(
+      "GetJobRequest{project_id=", request.project_id(),
+      ", job_id=", request.job_id(), ", location=", request.location(), "}");
+
+  EXPECT_EQ(expected, request.DebugString(TracingOptions{}.SetOptions(
+                          "truncate_string_field_longer_than=1024")));
+}
+
+TEST(ListJobsRequestTest, DebugString) {
+  auto const request = GetListJobsRequest();
+  std::string all_users = request.all_users() ? "true" : "false";
+  std::string expected = absl::StrCat(
```

Comment:
same optional comment about writing out the expected string instead of computing it.